Without this tf_to_kernel fails to link on AARCH64

Picked from https://github.com/tensorflow/tensorflow/commit/4933ada8c2b4b523acc94baea2609f0cce70a083

Author: Alexander Grund (TU Dresden)

diff --git a/tensorflow/core/platform/default/build_config.bzl b/tensorflow/core/platform/default/build_config.bzl
index d0dadd12108..bf6c631fdf6 100644
--- a/tensorflow/core/platform/default/build_config.bzl
+++ b/tensorflow/core/platform/default/build_config.bzl
@@ -795,9 +795,10 @@ def tf_google_mobile_srcs_only_runtime():
     return []
 
 def if_llvm_aarch64_available(then, otherwise = []):
-    # TODO(b/...): The TF XLA build fails when adding a dependency on
-    # @llvm/llvm-project/llvm:aarch64_target.
-    return otherwise
+    return select({
+        "//tensorflow:linux_aarch64": then,
+        "//conditions:default": otherwise,
+    })
 
 def if_llvm_system_z_available(then, otherwise = []):
     return select({
